### DISCUSSION TITLE
About units of until _ after _ sources

### DISCUSSION THREAD

**Discussion by mamiyaakira** [NanoComp/meep#2229] [Q&A] [answered]
The simulation was run with sim.run (until _ after _ sources = 100).
What are these 100 credits?

**Comment by FrockSaints** [comment, marked as answer]
These links should help:
https://meep.readthedocs.io/en/latest/Python_User_Interface/#run-and-step-functions
https://meep.readthedocs.io/en/latest/Introduction/#units-in-meep

**Reply by smartalecH** [reply]
The units used throughout meep (hint: it's whatever you define them to be) are thoroughly described here (reposting for improved clarity):

https://meep.readthedocs.io/en/latest/Introduction/#units-in-meep
